ecosystem/python/sdk/aptos_sdk/account.py

```diff
@@ -7,19 +7,17 @@
 import tempfile
 import unittest
 
-from . import ed25519
 from .account_address import AccountAddress
+from .ed25519 import PrivateKey, PublicKey
```

Comment:
It's funny cause I think this was changed the other way recently

Reply:
I prefer the other way for the key imports. It was intentional for if and when we sort other key types.

Reply:
I don't mind, I'll change it back.